event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java

```diff
@@ -32,4 +33,10 @@ void setSampleDefinitionFromJson(JsonNode data) {
   void setMetadataFromJson(JsonNode data) {
     this.metadata = data.toString();
   }
+
+  private List<SurveyFulfilment> allowedPrintFulfilments;
+
+  private List<SurveyFulfilment> allowedSmsFulfilments;
```

Comment:
is it possible to set these lists  in the `setSampleDefinitionFromJson(JsonNode data)` method above? That way we won't have to duplicate code in RHSvc and CCSvc to load these lists.